Update the documentation - Remove the "theme" key value pair from the block templates

imranhsayed opened the pull request:
## Description
- Removes the "theme" key value pair from the HTML comment of block templates, in the documentation.

As per the [recent changes in Gutenberg](https://github.com/WordPress/gutenberg/pull/28088) (as of 9.9) the "theme" attribute should be removed from the template-part blocks.

imranhsayed committed:
Update the documentation to remove the "theme" key value pair from the HTML comment

- Removes the "theme" key value pair from the HTML comment of block templates, in the documentation.

As per the [recent changes in Gutenberg](https://github.com/WordPress/gutenberg/pull/28088) (as of 9.9) the "theme" attribute should be removed from the template-part blocks.

carolinan commented on docs/designers-developers/developers/tutorials/block-based-themes/README.md:
````diff
@@ -172,15 +172,15 @@ Inside the block-templates folder, create an `index.html` file.
 ```
 
-If you used a different theme name, adjust the value for the theme key.
+If you used a block template part slug, adjust the value for the "slug" key.
````
I suggest deleting this, since this part is all about template parts.

carolinan commented on docs/designers-developers/developers/tutorials/block-based-themes/README.md:
````diff
@@ -172,15 +172,15 @@ Inside the block-templates folder, create an `index.html` file.
 ```
 <!-- wp:template-part {"slug":"header","theme":"myfirsttheme"} /-->
 
 <!-- wp:template-part {"slug":"footer","theme":"myfirsttheme"} /-->
````
```suggestion
<!-- wp:template-part {"slug":"footer"} /-->
```
Remove faulty code from the example

carolinan commented on docs/designers-developers/developers/tutorials/block-based-themes/README.md:
````diff
@@ -172,15 +172,15 @@ Inside the block-templates folder, create an `index.html` file.
+The HTML comments starts with `wp:template-part` which is the name of the template-part block type. Inside the curly brackets there is key and its values: The "slug" is the key and the slug of the template part e.g. "header", or "footer" is its value.
 
 ```
 <!-- wp:template-part {"slug":"header","theme":"myfirsttheme"} /-->
````
```suggestion
<!-- wp:template-part {"slug":"header"} /-->
```
Remove faulty code from the example

imranhsayed committed:
Apply suggestions from code review

Apply suggestions from code review

Co-authored-by: Carolina Nymark <myazalea@hotmail.com>

imranhsayed committed:
Remove unwanted line

remove unwanted line "If you used a block template part slug, adjust the value for the "slug" key."

imranhsayed commented:
Thank you @carolinan for the feedback. I have updated the PR as per your request.

imranhsayed commented:
@carolinan Please let me know if there are any other changes to be made.

carolinan commented:
This text is a bit difficult for me to read:
`Inside the curly brackets there is key and its values: The "slug" is the key and the slug of the template part e.g. "header", or "footer" is its value.`

imranhsayed committed:
Update the comment for the block template part slug and value

Update the comment for the block template part slug and value

imranhsayed commented:
Thanks @carolinan . I have made some changes as per your suggestions. Please let me know if it sounds better now.

carolinan commented:
Hi, can you refresh this?

ntsekouras commented:
Thanks for your work here @imranhsayed! I have to close this PR though as it seems this has been addressed by the massive docs update regarding the creation of a block theme here: https://github.com/WordPress/gutenberg/pull/31269/.

Thanks again!
